vis.mac

```diff
@@ -75,7 +78,8 @@
 # (14/03/10)
 # Factor of Normal PMT is 1.367
 /DarkRate/SetConvert 1.367
-#/DarkRate/SetConvert 1.120 #For HPDs
+#/DarkRate/SetConvert 1.120 #For HPDs (Detector: Cylinder_12inchHPD_15perCent and HyperK_withHPD)
+#/DarkRate/SetConvert 1.082 #For HPD_20150522 (Detector: SuperK_20inchHPD_20perCent, SuperK_20inchHPD_14perCent and SuperK_12inchHPD_15perCent)
```

Comment:
How is this conversion rate calculated? This number always seemed a bit mysterious to me.